### DISCUSSION TITLE
Comitup Issue Status for New Raspbian

### DISCUSSION THREAD

**Discussion by khaliduzzaman-mredul** [davesteele/comitup#257] [Q&A]
Is Comitup working with the latest version of raspbian now as initially it was having issues?

**Comment by davesteele** [comment]
To be clear, Raspbian is an obsolete OS (no updates in 4 years), though I believe Comitup has no issue with it. I'm running on the latest Raspbian Pi OS (Raspbios) with no problems.

I do need to update the image.

**Reply by khaliduzzaman-mredul** [reply]
That's great to know. But in my case, when I tried to install comitup using the .deb file it worked completely fine with my pi which I didn't update for like 1 year. But when I tried to install it on a newly installed pi it didn't work.

**Comment by davesteele** [comment]
I created a new image two days ago - it works for me. Deb installs require [more work](https://github.com/davesteele/comitup/wiki/Installing-Comitup).
